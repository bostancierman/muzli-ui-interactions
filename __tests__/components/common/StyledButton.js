import React from 'react'
import StyledButton from 'src/components/common/styled/StyledButton.js'

it('renders correctly', () => {
  expect(<StyledButton>Boilerplate</StyledButton>).toMatchSnapshot()
})
