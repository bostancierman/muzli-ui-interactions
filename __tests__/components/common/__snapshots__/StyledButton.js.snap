// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.button
  css={Immutable.Map {}}
>
  Boilerplate
</styled.button>
`;
